charts/sn-platform/values.yaml

```diff
@@ -230,8 +230,8 @@ images:
     repository: streamnative/pulsar_vault_init
     tag: "v1.0.6"
   custom_metric_server_prometheus:
-    repository: prom/prometheus
-    tag: "v2.39.1"
+    repository: quay.io/prometheus/prometheus
+    tag: "v2.41.0"
```

Comment:
> tag: "v2.24.0"
> tag: "v2.41.0"

Should we use two different versions of prometheus at the same time?

Reply:
No, should use same.

Reply:
updated in https://github.com/streamnative/charts/pull/977/commits/11c2901c4e6876c442b23524ab776ffa788bc761